lib/components/base/Checkbox.vue

```diff
@@ -25,43 +25,38 @@
 </template>
 <script setup>
 import { CheckIcon, DropdownIcon } from '@'
-</script>
-<script>
-import { defineComponent } from 'vue'
-
-export default defineComponent({
-  props: {
-    label: {
-      type: String,
-      default: '',
-    },
-    disabled: {
-      type: Boolean,
-      default: false,
-    },
-    description: {
-      type: String,
-      default: '',
-    },
-    modelValue: Boolean,
-    clickEvent: {
-      type: Function,
-      default: () => {},
-    },
-    collapsingToggleStyle: {
-      type: Boolean,
-      default: false,
-    },
+
+const emit = defineEmits(['update:modelValue'])
```

Comment:
```suggestion
const emit = defineEmits<{
	'update:modelValue': [boolean]
}>()
```

Use TypeScript to make this emit typed :)

Reply:
Done ^^.